docs/reference/builder.md

```diff
@@ -311,6 +311,12 @@ commands using a base image that does not contain `/bin/sh`.
 > If you want shell processing then either use the *shell* form or execute 
 > a shell directly, for example: `RUN [ "sh", "-c", "echo", "$HOME" ]`.
 
+> **Note**:
+> If you choose to use the *shell* form, any time you want to continue a single
+> `RUN` instruction onto the next line, it has to be ended with a backslash `\`.
+> For example,  `RUN /bin/bash -c 'source $HOME/.bashrc ;\` then on the next
+> line ` echo $HOME '`.
+
```

Comment:
@AmeBel Thank you for the contribution.  This doesn't need to be in a note.  As @thaJeztah there are already too many notes indicating structural issues here. So, take your great information out. Also, better to illustrate a line example with actual lines. 

---> https://gist.github.com/moxiegirl/fe122d248ff12654c1e7

 In the shell form you can use a `\` (backslash) to continue a single
 `RUN` instruction onto the next line. For example, consider these two lines:

```bash
RUN /bin/bash -c 'source $HOME/.bashrc ;\
echo $HOME'
```
Together they are equivalent to:

```bash
RUN /bin/bash -c 'source $HOME/.bashrc ; echo $HOME'
```